robottelo/ui/contentviews.py

```diff
@@ -130,65 +90,45 @@ def delete_version(self, name, version):
         element.click()
         self.wait_for_ajax()
         strategy, value = locators['contentviews.remove_ver']
-        remove_version = self.wait_until_element((strategy, value % version))
-        if remove_version is None:
-            raise UINoSuchElementError(
-                'Could not find button to delete version "{0}"'.format(version)
-            )
-        remove_version.click()
-        self.wait_for_ajax()
-        self.wait_until_element(
-            locators['contentviews.completely_remove_checkbox']
-        ).click()
-        self.wait_until_element(locators['contentviews.next_button']).click()
-        self.wait_for_ajax()
-        self.find_element(locators['contentviews.confirm_remove_ver']).click()
-        self.wait_for_ajax()
+        self.click((strategy, value % version))
+        self.click(locators['contentviews.completely_remove_checkbox'])
+        self.click(locators['contentviews.next_button'])
+        self.click(locators['contentviews.confirm_remove_ver'])
 
     def search(self, element_name):
-        """Uses the search box to locate an element from a list of elements """
-
+        """Uses the search box to locate an element from a list of elements"""
         element = None
-        strategy = locators["contentviews.key_name"][0]
-        value = locators["contentviews.key_name"][1]
-        searchbox = self.wait_until_element(common_locators["kt_search"])
+        strategy, value = locators['contentviews.key_name']
+        searchbox = self.wait_until_element(common_locators['kt_search'])
         if searchbox:
```

Comment:
I guess this condition should be refactored in some future PR too. It's low-priority nitpick, but still, if searchbox is missing on the page, we're returning `None` as if searchbox was ok and it's just element wasn't found. IMO, exception here would be really helpful for future test results analysis.